File: backend/src/main/java/com/campus/Campus_Connect/features/honor/dto/response/MedalResponse.java
package com.campus.Campus_Connect.features.honor.dto.response;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MedalResponse {

    private String title;

    private Boolean awarded;

    private MedalRecipientResponse recipient;

}

File: android-app/app/src/main/java/com/example/campusconnect/feature/events/data/remote/response/MedalResponse.kt
package com.example.campusconnect.feature.events.data.remote.response

import com.google.gson.annotations.SerializedName

data class MedalResponse(

    @SerializedName("title")
    val title: String,

    @SerializedName("awarded")
    val awarded: Boolean,

    @SerializedName("recipient")
    val recipient: MedalRecipientResponse?
)